tools/automation/cli_linter/linter.py

```diff
@@ -113,30 +114,36 @@ def run(self, run_params=None, run_commands=None, run_command_groups=None, run_h
                     found_rules.add(rule_name)
                     add_to_linter_func(self)
 
+        colorama.init()
         # run all rule-checks
         if run_help_files_entries and self._rules.get('help_file_entries'):
-            print(os.linesep + 'Running Linter on Help File Entries:')
             self._run_rules('help_file_entries')
 
         if run_command_groups and self._rules.get('command_groups'):
-            print(os.linesep + 'Running Linter on Command Groups:')
             self._run_rules('command_groups')
 
         if run_commands and self._rules.get('commands'):
-            print(os.linesep + 'Running Linter on Commands:')
             self._run_rules('commands')
 
         if run_params and self._rules.get('params'):
-            print(os.linesep + 'Running Linter on Parameters:')
             self._run_rules('params')
 
+        if not self.exit_code:
+            print('\nNo violations found.')
+        colorama.deinit()
         return self.exit_code
 
     def _run_rules(self, rule_group):
-        for callable_rule in self._rules.get(rule_group):
-            for violation_msg in sorted(callable_rule()):
-                print(violation_msg)
-            print()
+        from colorama import Fore
+        for rule_name, rule_func in self._rules.get(rule_group).items():
+            violations = sorted(rule_func()) or []
+            if violations:
+                print('- {} FAIL{}: {}'.format(Fore.RED, Fore.RESET, rule_name))
+                for violation_msg in violations:
+                    print(violation_msg)
+                print()
+            else:
+                print('- {} pass{}: {} '.format(Fore.GREEN, Fore.RESET, rule_name))
```

Comment:
kinda nitpicky but maybe only one or zero spaces? or just get rid of the string, "-  ", before the `pass/FAIL`. Feel free to ignore as well.